TradeItIosTicketSDK2/TradeItPortfolioAccountsTableViewManager.swift

```diff
@@ -203,10 +203,13 @@ fileprivate class LinkedBrokerSectionPresenter {
     private func errorCell(forTableView tableView: UITableView) -> UITableViewCell {
         guard let error = error else { return UITableViewCell() }
         let cell = tableView.dequeueReusableCell(withIdentifier: "TRADE_IT_PORTFOLIO_LINKED_BROKER_ERROR") ?? UITableViewCell(style: .subtitle, reuseIdentifier: nil)
-        if error.requiresRelink() == true {
+        if error.isAccountLinkDelayedError() {
+            cell.textLabel?.text = "Activation in Progress"
+            cell.detailTextLabel?.text = "Account is being activated. Tap to retry."
```

Comment:
Let's review the wording here. If it is going to take 1-2 days then the user might get confused by telling them to `Tap to retry`.